.github/workflows/static-analysis.yaml

```
@@ -19,7 +19,7 @@ jobs:
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
-          php-version: '8.1'
+          php-version: '8.3'
```

This is failing, and PHP-CS-Fixer is slow to support newer language versions. We should stick to 8.2 here
```suggestion
          php-version: '8.2'
```

---

>PHP-CS-Fixer is slow to support newer language versions

It makes me sad, but I agree. I wanted to run the tol on PHP `8.3` just for the sake of consistency with the other jobs, but I remember that in the past we had to wait on average a long time for a compatible version to be released